docs/03-customization.md

```diff
@@ -17,19 +17,18 @@ You can read more about environment variables [here](https://www.balena.io/docs/
 
 The following environment variables apply to balenaSound in general, modifying it's behaviour across the board:
 
-| Environment variable | Description | Options | Default | 
+| Environment variable | Description | Options | Default |
 | ------ | ------ | ------ | ------ |
 | SOUND_MODE | Select which mode of operation a device should use. For a detailed explanation see [here](../docs/usage#modes-of-operation).| `MULTI_ROOM`, `MULTI_ROOM`, `STANDALONE` | `MULTI_ROOM`, except for cases listed [here](../docs/device-support). |
 | SOUND_VOLUME | Output volume level at startup. | 0 - 100, integer value without the `%` symbol. | 75 |
-| SOUND_DEVICE_NAME | Device name to be advertized by plugins (AirPlay device list, Spotify Connect, bluetooth search list, etc). | Any valid string. | `balenaSound <plugin> <xxxx>`, where:<br>- `<plugin>` is `Spotify, AirPlay, UPnP`<br>- `<xxxx>` the first 4 chars of the device UUID. |
```

Comment:
I added back the `<br>` because it's the only way to have landr render this section in a way that looks decent. Without it it's just a very long chain of strings.